pages/index.tsx

```diff
@@ -34,7 +34,6 @@ export default function Home({ events, filters }) {
 
   Refer to https://github.com/cesium/calendarium/pull/39
```

Comment:
If you are removing this, remove the above comment too 😅

Reply:
true